ob1k-cache/src/main/java/com/outbrain/ob1k/cache/LocalAsyncCache.java

```diff
@@ -114,7 +118,8 @@ public LocalAsyncCache() {
       return fromValue(res);
     } else {
       if (failOnMissingEntries) {
-        return fromError(new RuntimeException("result is missing from loader response."));
+        final String prefix = cacheName == null ? "" : cacheName + ": ";
+        return fromError(new RuntimeException(prefix + "result for " + key + " is missing from loader response."));
```

Comment:
`key` is Object, which may, or may not, implement toString.
just append cacheName, will be enough.